torch/__init__.py

```diff
@@ -74,7 +74,7 @@
     import glob
     dlls = glob.glob(os.path.join(th_dll_path, '*.dll'))
     for dll in dlls:
-        ctypes.CDLL(lib_path)
+        ctypes.CDLL(dll)
```

Comment:
Do you know when CDLL unloads the library? When GC collects a variable?
```suggestion
        ctypes.LoadLibrary(dll)
```

Reply:
The library will not be unloaded automatically. We need to manually do that by calling the win32 api FreeLibrary.

Reply:
About the suggested code change, I checked the source code of `ctypes` and they are just the same, except that CDLL specifies the mode to be `nt._LOAD_LIBRARY_SEARCH_DEFAULT_DIRS | nt._LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR`.